lightning/src/ln/chan_utils.rs

```diff
@@ -587,24 +600,24 @@ impl LocalCommitmentTransaction {
 			},
 			their_sig: dummy_sig,
 			our_sig_first: false,
-			local_keys: TxCreationKeys {
+			holder_keys: TxCreationKeys {
```

Comment:
Huh, should this just be `keys` instead? Its awkward to have a holder_keys.countersignatory_htlc_keys field.